x-pack/plugins/screenshotting/server/browsers/extract/extract_error.ts

```diff
@@ -6,9 +6,9 @@
  */
 
 export class ExtractError extends Error {
-  public readonly cause: string;
+  public readonly cause: Error;
```

Comment:
Cause is now a property on the base `Error` class so sub-classes must use type `Error` instead of string. Conveniently one of the two uses of `ExtractError` was already passing an error as the `cause`.